repairnator/src/main/java/fr/inria/spirals/repairnator/process/ProjectInspector.java

```diff
@@ -26,22 +26,22 @@
  * Created by urli on 26/12/2016.
  */
 public class ProjectInspector {
-    private final Logger logger = LoggerFactory.getLogger(ProjectInspector.class);
+    protected final Logger logger = LoggerFactory.getLogger(ProjectInspector.class);
```

Comment:
I don't agree: Loggers are private to their class you should not use inheritance of logger, but create your own in the subclass.